src/ui/ContextMenu/items/EmojiListItems.jsx

```diff
@@ -18,11 +18,13 @@ export default class EmojiListItems extends Component {
     this.setupEvents();
     this.getBarPosition();
     this.showParent();
+    document.body.style.overflow = 'hidden';
   }
 
   componentWillUnmount() {
     this.cleanUpEvents();
     this.hideParent();
+    document.body.style.overflow = 'unset';
```

Comment:
Not sure this is a good idea to modify customer's document style

Reply:
Better thing to do is, add a class to body, and set overflow as "hidden !important" and remove class after unmount